### PR TITLE
fixed demigod 2+3 and harphy bubbles

### PR DESCRIPTION
**Pull request by Nemoel-Tomo**

### CODE REVIEW

File: TombEngine/Objects/TR4/Entity/tr4_harpy.cpp [hunk 1]
```diff
@@ -225,15 +225,15 @@ namespace TEN::Entities::TR4
 			if (item->ItemFlags[0] <= 65 && GlobalCounter & 1)
 			{
 				auto pos3 = GetJointPosition(item, HarpyAttack1.meshNum, Vector3i(HarpyAttack1.Position.x, HarpyAttack1.Position.y * 2, HarpyAttack1.Position.z));
-				auto orient = Geometry::GetOrientToPoint(rh.ToVector3(), lr.ToVector3());
+				auto orient = Geometry::GetOrientToPoint(lr.ToVector3(), rh.ToVector3());//rh  lr
```
Comment (Lwmte) [inline TombEngine/Objects/TR4/Entity/tr4_harpy.cpp:228]: Please remove garbage comment here

File: TombEngine/Objects/TR4/Entity/tr4_demigod.cpp [hunk 2]
```diff
@@ -159,7 +159,7 @@ namespace TEN::Entities::TR4
 			auto* fx = &EffectList[fxNumber];
 
 			fx->pos.Position.x = pose->Position.x;
-			fx->pos.Position.y = pose->Position.y - (GetRandomControl() & 0x3F) - 32;
+			fx->pos.Position.y = pose->Position.y - (GetRandomControl() & 0x3F) - 32; //war -
```
Comment (Lwmte) [inline TombEngine/Objects/TR4/Entity/tr4_demigod.cpp:162]: Please remove garbage comment here

File: TombEngine/Objects/TR4/Entity/tr4_demigod.cpp [hunk 3]
```diff
@@ -171,6 +171,7 @@ namespace TEN::Entities::TR4
 
 			fx->pos.Orientation.z = 0;
 
+			fx->fallspeed = 0;
```
Comment (Lwmte) [inline TombEngine/Objects/TR4/Entity/tr4_demigod.cpp:174]: Does this value (gravity) also provokes incorrect bubble behaviour?

File: TombEngine/Objects/TR4/Entity/tr4_harpy.cpp (hunk 1 above)
Comment (Nemoel-Tomo) [reply TombEngine/Objects/TR4/Entity/tr4_harpy.cpp]: fixed

File: TombEngine/Objects/TR4/Entity/tr4_demigod.cpp (hunk 2 above)
Comment (Nemoel-Tomo) [reply TombEngine/Objects/TR4/Entity/tr4_demigod.cpp]: fixed

File: TombEngine/Objects/TR4/Entity/tr4_demigod.cpp (hunk 3 above)
Comment (Nemoel-Tomo) [reply TombEngine/Objects/TR4/Entity/tr4_demigod.cpp]: removed

**Pull request merged**